samples/buildrun/buildrun_buildah_cr.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
 spec:
   buildRef:
     name: buildah-golang-build
+  serviceAccount: default
```

Comment:
Is this needed for tests? If yes, could we dynamically add it, please ? Otherwise, it would break the low-touch usage on openshift :)

Reply:
This new change is not for this feature. I agree to dynamically add it.

I remove this change in this PR first and we can refine our e2e to dynamically add in future